docs/02-for-app-authors/03-appdata-guidelines/index.md

```diff
@@ -1,14 +1,16 @@
 # AppData guidelines
 
-These aren’t necessarily _requirements_, those are covered in [requirements](/docs/for-app-authors/requirements#appstream). These are more tips/best practices to help you get your AppData up to spec.
+These are a set of guidelines for AppData that should be followed for submission on Flathub.
```

Comment:
So the difference between https://docs.flathub.org/docs/for-app-authors/submission/ and this now is that this page is about appdata only?

Reply:
The submission page doesn't have anything about appdata, are you talking about the requirements page?

Reply:
No, I was thinking about how we guide people toward the docs. flathub.org links to the submission page directly right now, so mandatory things to know on pages before that might be weird. That's all I'm asking/thinking about. (not sure it was a solved problem before, we always struggled with this IMO)

Reply:
I think the submission page has something asking to check the requirements page and requirements page links this page in the appdata header.

Reply:
Yeah, I guess linking is fine then